### PR TITLE
Men xxx add menelaos front to repo

### PR DESCRIPTION
**Pull request by luksmi**

### CODE REVIEW

File: menelaos-front/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Editor configuration, see https://editorconfig.org
```
Comment (mkowalc7) [inline menelaos-front/.editorconfig]: Should be removed

File: menelaos-front/README.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
+# MenelaosFront
```
Comment (mkowalc7) [inline menelaos-front/README.md]: Unnecessary file, should be removed. The content probably could be moved to ../README.md

File: menelaos-front/.editorconfig (hunk 1 above)
Comment (luksmi) [reply menelaos-front/.editorconfig]: By generating .gitignore file, Angular says it shouldn't

File: menelaos-front/README.md (hunk 2 above)
Comment (luksmi) [reply menelaos-front/README.md]: Done

**Comment by mkowalc7** [comment]
Thanks.

**Pull request merged**